### DISCUSSION TITLE
求一个linux版巫师2游戏

### DISCUSSION THREAD

**Discussion by jimross333** [Gamuxorg/bbs#353] [投递和请求游戏]
求一个linux版巫师2 appimage

**Comment by shouhuanxiaoji** [comment]
这个游戏在linux上并非原生制作，用的是类似wine的技术，叫eON，目前运行起来效率较差，我之前是4g内存+nvidia 660gtx勉强运行，a卡更惨不忍睹，你确定要试试？

**Comment by shiptux** [comment]
这个游戏我曾经在linux下运行过，rx470d中效稍微卡顿，优化不是很好建议不要在linux下运行。

**Comment by shouhuanxiaoji** [comment]
有依赖问题

**Comment by wangling12** [comment]
我前一段时间上传了一个,你可以试试
